File: fixtures/projects/generic/swift/src/First.swift
struct FirstChoice {
  func choose(_ left: Int, _ right: Int) -> Int {
    var total = left + right
    let limit = 10
    total = total * 2
    if left > 0 && right != 0 {
      total = total + limit
    }
    return total
  }
}

File: fixtures/projects/generic/rust/src/first.rs
pub fn primary_choice(left: i32, right: i32) -> i32 {
    let mut total = left + right;
    let limit = 10;
    total = total * 2;
    if left > 0 && right != 0 {
        total = total + limit;
    }
    total
}
